miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -5,19 +5,17 @@ package com.rakuten.tech.mobile.miniapp
  * @property baseUrl Base URL used for retrieving a Mini App.
  * @property rasAppId App ID for the Platform API.
  * @property subscriptionKey Subscription Key for the Platform API.
- * @property hostAppVersionId Version of the host app, used to determine feature compatibility for Mini App.
  * @property hostAppUserAgentInfo User Agent information from Host App.
  * @property isTestMode Whether the sdk is making use of the Test API Endpoints for under "Testing" mini apps.
  */
 data class MiniAppSdkConfig(
     val baseUrl: String,
     val rasAppId: String,
     val subscriptionKey: String,
-    val hostAppVersionId: String,
```

Comment:
I think we wanted to keep the `hostAppVersionId` setting, but just remove it from the API requests. This is because we might want to use it in the future for returning different versions of a mini app based on the version of the Host App (that was the settings original purpose, but the functionality was never added).

I guess we could remove it, but this is a breaking change since `MiniAppSdkConfig` is public. So we would need to just deprecate this parameter instead, so this gets a little complicated.

What do you think, should we deprecate it or just leave the setting here and not use it for anything?

Reply:
We can leave this setting here. This PR implementation causes no impact whether the user defines the manifest of hostapp version or not.